README.md

```diff
@@ -17,6 +17,12 @@ This is a PHP library for MediaWiki extensions. It does not in itself add or enh
 * [PHP](http://php.net/) 5.3 or later (including PHP 7)
 * [MediaWiki](https://www.mediawiki.org/) 1.16 or later
 
+The latest versions (1.6+) requires more recent PHP and MediaWiki. If that does not work for you, just
+stick with version 1.5, which has no known defects and does not miss any real features. 
+
+* [PHP](http://php.net/) 5.5 or later (including PHP 7.x)
+* [MediaWiki](https://www.mediawiki.org/) 2.27 or later
```

Comment:
Time travel bunny does not find such a software

![timebunny](https://cloud.githubusercontent.com/assets/146040/24390103/da1c3e3e-1386-11e7-86f2-e9dc5801288f.jpg)